assets/docs-theme.mjs

```diff
@@ -0,0 +1,38 @@
+import { copyFileSync } from 'fs';
```

Comment:
It's been a minute since I've had to deal with commonjs vs. modules vs. ts config. I'm not sure if an `.mjs` file was the best way to do this, and I also wasn't sure if there were specific considerations around adding a script that's outside `/src/` and used in CI / build. Definitely open to changing things up here.